src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
@ljharb I can convert these to functions. Would that be acceptable? Want to make sure because it's a more involved change. 😄  I understand it would have to wait for a 3.x release.

Are there any concerns about ReactWrapper having properties for these and ShallowWrapper having functions?